ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -398,6 +398,10 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    public override func viewSafeAreaInsetsDidChange() {
+        needsOffsetUpdate = true
```

Comment:
Is it worth calling super for safety? In case our inheritance changes?

Reply:
Sure, makes sense.